### PR TITLE
Student distributions

### PR DESCRIPTION
**Pull request by charliekelly13**
Completed stats distribution back end

### CODE REVIEW

File: src/main/java/universityWebApp/model/Staff.java [hunk 1]
```diff
@@ -1,4 +1,99 @@
...
+
+    public String getNationality() {
+        return nationality;
+    }
+
+    public void setNationality(String nationality) {
+        this.nationality = nationality;
+    }
+
+//    public String getAddress() {
```
Comment (DarraghClarke) [inline src/main/java/universityWebApp/model/Staff.java]: do we want this commented in, or we can delete

File: src/main/java/universityWebApp/controller/StatsController.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            }
+        }
+        return genderDis;
+    }
+
+    /**
+     * This endpoint returns all distribution of staff nationality
+     */
+    @RequestMapping(value = "staff", method = RequestMethod.GET)
+    public HashMap getNationalityDistribution() {
```
Comment (DarraghClarke) [inline src/main/java/universityWebApp/controller/StatsController.java]: should return string of the .jsp file

File: src/main/java/universityWebApp/model/Staff.java [hunk 3]
```diff
@@ -1,4 +1,99 @@
...
+    public Staff() {
+        super();
+    }
+
+    public Staff(String id, String name, String gender) {
+        super();
+        this.id = id;
+        this.name = name;
+        this.gender = gender;
+//        this.address = address;
```
Comment (oisinq) [inline src/main/java/universityWebApp/model/Staff.java]: Can we delete the commented code?

File: src/main/java/universityWebApp/controller/StatsController.java (hunk 2 above)
Comment (oisinq) [reply src/main/java/universityWebApp/controller/StatsController.java]: Yeah, we want to combine these two methods in one /stats endpoint

File: src/main/java/universityWebApp/controller/StatsController.java [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+import java.util.List;
+
+public class StatsController {
+    StaffRepository staffRepository;
+    StudentRepository studentRepository;
+    /**
+     * This endpoint returns all distribution of user gender
+     */
+    @RequestMapping(value = "stats", method = RequestMethod.GET)
+    public String getGenderDistribtuion(Model model) {
```
Comment (DarraghClarke) [inline src/main/java/universityWebApp/controller/StatsController.java]: could you take the two methods getGenderDistribtuion and getNationalityDistribtuion and combine them into one method , and then have them just add their two indiviual results to the model like you are atm, then it'll be ready I think

File: src/main/java/universityWebApp/model/Staff.java (hunk 3 above)
Comment (charliekelly13) [reply src/main/java/universityWebApp/model/Staff.java]: Yes

**Pull request merged**